unstructured_ingest/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.24"  # pragma: no cover
+__version__ = "0.0.25"  # pragma: no cover
```

Comment:
Change to 0.0.26-dev0

Reply:
Ha. You'll have to keep updating it until it is beyond what is in Main. We need to see the CI tests run when you push your code. :) One of the "fun" parts of github.